crates/gem_evm/src/uniswap/deployment/mod.rs

```diff
@@ -59,5 +59,10 @@ pub fn get_provider_by_chain_contract(chain: &Chain, contract: &str) -> Option<S
             return Some(SwapProvider::Reservoir.id().to_string());
         }
     }
+    if let Some(deployment) = v3::get_aerodrome_router_deployment_by_chain(chain) {
+        if deployment.universal_router.to_lowercase() == contract.to_lowercase() {
```

Comment:
```suggestion
        if deployment.universal_router.to_lowercase() == contract.to_lowercase() {
```

maybe avoid lowecase? always store in checksum values

Reply:
i'd like to keep as it is